lms/djangoapps/learner_dashboard/views.py

```diff
@@ -77,11 +78,19 @@ def program_details(request, program_uuid):
 
     if waffle.switch_is_active('new_program_progress'):
         course_progress = meter.progress(programs=[program_data], count_only=False)[0]
+
+        sidebar_data = {
+            # TODO: Reuse course_progress?
+            'progress': meter.progress(programs=[program_data])[0]
```

Comment:
yeah imo it seems wasteful to me to make the progress call twice, think just checking the length would be fine